tools/topology/sof-apl-src-50khz-pcm512x.m4

```diff
@@ -0,0 +1,72 @@
+#
+# Topology for Apollolake UP^2 with pcm512x codec with sample rate
+# conversion (SRC component).
+#
+
+# Include topology builder
+include(`utils.m4')
+include(`dai.m4')
+include(`pipeline.m4')
+include(`ssp.m4')
+
+# Include TLV library
+include(`common/tlv.m4')
+
+# Include Token library
+include(`sof/tokens.m4')
+
+# Include Apollolake DSP configuration
+include(`platform/intel/bxt.m4')
+
+DEBUG_START
+
+#
+# Define the pipelines
+#
+# PCM0 ----> src -----> SSP5 (pcm512x)
+#
+
+dnl PIPELINE_PCM_ADD(pipeline,
+dnl     pipe id, pcm, max channels, format,
+dnl     frames, deadline, priority, core)
+
+# Playback pipeline 1 on PCM 0 using max 2 channels of s24le.
+# Schedule 48 frames per 1000us deadline on core 0 with priority 0
```

Comment:
Do we also need to change the comments here?
PS: I think this commit should go after the `topology: enable pcm range and pipeline rate and remove frame count`

Reply:
ok